components/segmenter/tests/testdata/LineBreakExtraTest.txt

```diff
@@ -0,0 +1,6 @@
+# Additional line breaking tests, not in LineBreakTest.txt
```

Comment:
Are these hand-written or sourced from somewhere? Please add to the comment and link the PR for context

Reply:
This is hand-writing. SentenceBreak test has same way to add new test excepting to Unicoide tools'  test.